### PR TITLE
Work backend

### PR DESCRIPTION
**Pull request by csh3695**
# Fix Needed
### AdReception
- AdReception view created. but still redirection not working.
- Making random link implemented by [hashids](https://hashids.org/python/). Maybe we need more discussion using this.
#### AditUser
- Point field needed
- Is pw changeable..? (->maybe on ver2.0)

#
# Thx:)

### CODE REVIEW

**Comment by DHdroid** [comment]
Nice Work!

File: aditback/adit/views.py [hunk 1]
```diff
@@ -231,32 +231,85 @@ def get(self, request, id):
...
+                tag_new = InterestedTags(content = tag, usercount = 1, postcount = 0)
+                tag_new.save()
+                adpost.tags.add(tag_new)
+
+        adpost.save()
+
+        for i in range(len(post_new_images)):
+            if i>0:
+                newimg = img_process(post_new_images[i])
+                adpost.image.add(newimg)
```
Comment (sanggggg) [inline aditback/adit/views.py]:
```python
for img in post_new_images[1:]:
    ...
````

가 어떤가여

**Comment by digdhg** [comment]
adreception/redirect/ 부분 wiki에 update해주시면 감사하겠습니다.
LGTM

File: aditback/adit/views.py [hunk 2]
```diff
@@ -231,32 +231,85 @@ def get(self, request, id):
...
+        for image in post_old_images:
+            PostImage.delete(image)
+        adpost.image.clear()
+
+        img_new = img_process(post_new_thumbnail)
+        adpost.thumbnail = img_new
+        #PostImage.delete(post_old_thumbnail)
+
+        for tag in post_new_tags:
+            if len(InterestedTags.objects.filter(content = tag)) > 0:
```
Comment (sanggggg) [inline aditback/adit/views.py]:
```python
    ...
    if InterestedTags.objects.filter(content=tag).exists():
    ...
```

len하면 저거 다 쿼리 보내는 거라 exist가 빠를거에ㅕ

File: aditback/adit/views.py (hunk 1 above)
Comment (csh3695) [reply aditback/adit/views.py]: Done.

File: aditback/adit/views.py (hunk 2 above)
Comment (csh3695) [reply aditback/adit/views.py]: Done.

**Pull request merged**